e2e/pages/organizations/developers/developers.spec.js

```diff
@@ -41,6 +47,133 @@ describe('the Developers page', () => {
         it('should not have friendly name text box under POC', () => {
           expect(contact.friendlyName.isDisplayed()).toBe(false);
         });
+
+        it('should not allow to edit developer status', () => {
+          expect(page.developerStatus.isDisplayed()).toBe(false);
+        });
+
+        it('should allow editing of POC', () => {
+          let timestamp = (new Date()).getTime();
+          let poc = {
+            full: 'name' + timestamp,
+            title: 'title' + timestamp,
+            email: 'email' + timestamp + '@example.com',
+            phone: 'phone' + timestamp,
+            website: 'https://website' + timestamp + '.com',
+          };
+          contact.set(poc);
+          actionBar.save();
+          expect(toast.toastTitle.getText()).toEqual('Update processing');
+          hooks.waitForSpinnerToDisappear();
+          expect(page.developerContact).toHaveTextContaining(poc.full);
+          expect(page.developerContact).toHaveTextContaining(poc.title);
+          expect(page.developerContact).toHaveTextContaining(poc.phone);
+          expect(page.developerContact).toHaveTextContaining(poc.email);
+          expect(page.developerWebsite).toHaveTextContaining(poc.website);
+        });
+
+      });
+      describe('when looking at developer with more than one product', () => {
+
+        it('should have split developer button', () => {
+          expect(page.splitDeveloper.isDisplayed()).toBe(true);
+        });
+        it('should not have merge developer button', () => {
+          expect(page.mergeDeveloper.isDisplayed()).toBe(false);
+        });
+
+        it('should allow split to happen', () => {
+          page.splitDeveloper.click();
```

Comment:
The spacing between blocks here is inconsistent
```suggestion
        });
      });

      describe('when looking at developer with more than one product', () => {
        it('should have split developer button', () => {
          expect(page.splitDeveloper.isDisplayed()).toBe(true);
        });

        it('should not have merge developer button', () => {
          expect(page.mergeDeveloper.isDisplayed()).toBe(false);
        });

        it('should allow split to happen', () => {
          page.splitDeveloper.click();
```